packages/grafana-ui/src/types/panel.ts

```diff
@@ -37,7 +37,11 @@ export interface PanelProps<T = any> {
 
 export interface PanelEditorProps<T = any> {
   options: T;
-  onOptionsChange: (options: T) => void;
+  onOptionsChange: (
+    options: T,
+    // callback can be used to run something right after update.
+    callback?: () => void
```

Comment:
Is this necessary? I can't see any callback being passed to this prop anywhere in this PR.

Reply:
It's passed here https://github.com/grafana/grafana/pull/19804/files#diff-f715345ae09960ce639d3e8daf07aff6R56 and this is what is passed as `onOptionsChange` https://github.com/grafana/grafana/pull/19804/files#diff-6e9276e76a52b87735ffbbb4306fa5a8R168